backend/tests/api/test.env

```diff
@@ -0,0 +1,8 @@
+PROMETHEUS_MULTIPROC_DIR=/tmp
+# DB connection
+POSTGRES_USER=postgres-test-user
+POSTGRES_PASSWORD=postgres-test-pw
+POSTGRES_DB=postgres-test-db
+POSTGRES_PORT=5433
```

Comment:
why do we need a different port number it was 5434

Reply:
Hi @robela - just to be clear, this is not a PR into main. It's into a feature branch so that we can incrementally build features, get feedback from each other, while keeping main stable.

Branches outside of main are not protected by reviews. Do you want to be approving PRs into non-main branches?